ilc/client/TransactionManager.js

```diff
@@ -136,20 +136,19 @@ export class TransactionManager {
     };
 
     #runGlobalSpinner = () => {
-        const registrySettings = this.#registrySettings;
-        if (registrySettings.globalSpinner === false || this.#spinnerTimeout) {
+        if (this.#spinnerConfig === false || this.#spinnerTimeout) {
             return;
         }
 
         this.#spinnerTimeout = setTimeout(() => {
-            if (registrySettings.globalSpinner === true) {
+            if (this.#spinnerConfig === true) {
```

Comment:
do we have any ability to set boolean to spinnerConfig?

Reply:
Not sure that I understood you... pls elaborate

Reply:
sorry, my mistake, I read incorrectly this condition https://github.com/namecheap/ilc/blob/feature/i18n/global_spinner_config_to_registry/ilc/client/TransactionManager.js#L23